src/tools/rbd_mirror/image_replayer/journal/Replayer.h

```diff
@@ -97,6 +97,9 @@ class Replayer : public image_replayer::Replayer {
     return m_image_spec;
   }
 
+  void prune_snapshot(uint64_t snap_id) {
```

Comment:
Consider a different function signature - snapshots are not relevant to journal based mirroring.